system/Session/Handlers/RedisHandler.php

```diff
@@ -22,6 +22,8 @@
  */
 class RedisHandler extends BaseHandler
 {
+    private const REDIS_DEFAULT_PORT = 6379;
```

Comment:
```suggestion
    private const DEFAULT_PORT = 6379;
```

What about `RedisHandler::DEFAULT_PORT`?

Reply:
Yeah, `REDIS_` is not needed.
Done.